test/checkImageSize.js

```diff
@@ -1,86 +1,26 @@
-const sizeOf = require('image-size');
-const fs = require('fs');
-const { exit } = require("process");
+import "process";
+import {imageSizeChecker} from './library.js';
+import {getAllFiles} from './library.js';
```

Comment:
Let's keep spacing consistent for the entire project: 

```suggestion
import { imageSizeChecker } from './library.js';
import { getAllFiles } from './library.js';
```